app/services/report_errors_to_argo.rb

```diff
@@ -0,0 +1,24 @@
+class ReportErrorsToArgo
+  WORKFLOW = 'preservationWF'.freeze
+  WORKFLOW_DEF = <<-EOXML
+    <workflow id="preservationWF">
+         <process name="start" />
+         <process name="moab-valid"  />
+         <process name="M2C-check-exist" />
+         <process name="C2M-check-exist" />
+    </workflow>
+    EOXML
+  REPO = "dor".freeze
+
+  def initialize(druid)
+    @druid = druid
+  end
+
+  def send_errors(workflow_process, error_msg)
+    unless Dor::WorkflowService.get_active_workflows(REPO,"druid:#{@druid}").include?(WORKFLOW)
```

Comment:
ugh - seems sound; wondering about performance.  Guess we'll find out!  Will we need newrelic to help us with knowing wait times for network call?

Reply:
Good point @ndushay -- we turned newrelic off because it didn't register seeding (and such) activity. I think the monitoring it offered us was pretty controller-centric, though, so it's not clear it would help us out in this case, but maybe benchmarking will?